providers/base/units/input/test-plan.pxu

```diff
@@ -32,8 +33,9 @@ unit: test plan
 _name: Input tests (after suspend)
 _description: Input tests (after suspend)
 include:
-    suspend/pointing-after-suspend_.*          certification-status=blocker
-    suspend/clicking-after-suspend_.*          certification-status=blocker
+    after-suspend-input/pointing_.*                          certification-status=blocker
+    after-suspend-input/clicking_.*                          certification-status=blocker
+    after-suspend-input/keyboard                             certification-status=blocker
```

Comment:
If `input/accelerometer` is added back, then there should be a `after-suspend-input/accelerometer` here as well.

Reply:
Added after-suspend-input/accelerometer in the test plan